sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -231,4 +233,28 @@ export class ApplicationAESTController extends BaseController {
       applicationId,
     );
   }
+
+  /**
+   * Get history of application versions for an application
+   * where the current application is excluded.
+   * @param applicationId application Id.
+   * @returns history of application versions.
+   */
+  @ApiNotFoundResponse({
+    description: `Application not found or not on ${ApplicationStatus.Completed} status.`,
+  })
+  @Get(":applicationId/versions")
+  async getApplicationVersionHistory(
+    @Param("applicationId", ParseIntPipe) applicationId: number,
+  ): Promise<ApplicationVersionAPIOutDTO[]> {
+    const applications =
+      await this.applicationService.applicationVersionHistory(applicationId);
+    if (!application) {
+      throw new NotFoundException("Application not found.");
+    }
+    return applications.map((application) => ({
+      id: application.id,
+      submittedDate: getPSTPDTDateHourMinute(application.submittedDate),
```

Comment:
Was there a requirement to convert the date to PDTPST? Let me know if I am missing something.

I believe we should not convert the date to PDTPST and send only the raw date object from API. 

It can be formatted at Vue to desired date/date-time format.